components/platform/platform.c

```diff
@@ -311,6 +402,57 @@ int platform_uart_start( unsigned id )
   if(uart_event_task_id == 0)
     uart_event_task_id = task_get_id( uart_event_task );
 
+#if CONFIG_ESP_CONSOLE_USB_SERIAL_JTAG
+  if (id == 0) {
+    if (usbcdc_event_task_id == 0) {
+      usbcdc_event_task_id = task_get_id(usbcdc_event_task);
+
+
+      /* Disable buffering on stdin */
+      setvbuf(stdin, NULL, _IONBF, 0);
+
+      /* Minicom, screen, idf_monitor send CR when ENTER key is pressed */
+      esp_vfs_dev_usb_serial_jtag_set_rx_line_endings(ESP_LINE_ENDINGS_CR);
```

Comment:
Should this be a menuconfig option perhaps? What does esplorer send?

Reply:
This is also what esplorer sends -- I copied this bit of code from the idf along with the comment!